test/src/tests-integration-capture.ts

```diff
@@ -71,13 +72,11 @@ describe('Integration Capture', () => {
         expect(testEvent.data).to.have.property('event_name', 'Test Event');
         expect(testEvent.data).to.have.property('custom_flags');
 
-        expect(testEvent.data.custom_flags).to.deep.equal({
-            'Facebook.ClickId': `fb.1.${initialTimestamp}.1234`,
-            'Facebook.BrowserId': '54321',
-            'GoogleEnhancedConversions.Gclid': '234',
-            'GoogleEnhancedConversions.Gbraid': '6574',
-            'GoogleEnhancedConversions.Wbraid': '1234111',
-        });
+        expect(testEvent.data.custom_flags['Facebook.ClickId'], 'Facebook Click Id').to.equal(`fb.1.${initialTimestamp}.1234`);
+        expect(testEvent.data.custom_flags['Facebook.BrowserId'], 'Facebook Browser Id').to.equal('54321');
+        expect(testEvent.data.custom_flags['GoogleEnhancedConversions.Gclid'], 'Google Enhanced Conversions Gclid').to.equal('234');
+        expect(testEvent.data.custom_flags['GoogleEnhancedConversions.Gbraid'], 'Google Enhanced Conversions Gbraid').to.equal('6574');
+        expect(testEvent.data.custom_flags['GoogleEnhancedConversions.Wbraid'], 'Google Enhanced Conversions Wbraid').to.equal('1234111');
```

Comment:
any reason for the formatting change?  should you add roktid to this

Reply:
Not exactly. This is to make it easier to diagnose failing tests when a specific click ID is broken or missing. I realized when we were using an object comparison, Karma/Chai wasn't really reporting which specific click ID was missing.